client/components/popupbox/popupbox-directive.js

```diff
@@ -183,12 +207,19 @@ explorer.components.popupbox.PopupboxDirective = function($timeout) {
         scope.hidePopup();
       };
 
+      /**
+       * Returns true if the displayValue starts with the current text,
+       * otherwise false.
+       */
+      scope.startsWith = function(data) {
+        let displayValue = scope.getDisplayValue(data);
+
+        return (displayValue.indexOf(input[0].value) == 0);
```

Comment:
Consider [goog.string.startsWith](http://closure-library.googlecode.com/git-history/docs/namespace_goog_string.html) for clarity.